### DISCUSSION TITLE
Welcome to Swift-Async-Await-Experiments Discussions!

### DISCUSSION THREAD

**Discussion by peterfriese** [peterfriese/Swift-Async-Await-Experiments#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ejstembler** [comment]
Really Iiked your [blog post](https://peterfriese.dev/async-await-in-swiftui)!  I'm guessing the [latest trunk](https://swift.org/builds/development/xcode/swift-DEVELOPMENT-SNAPSHOT-2021-03-02-a/swift-DEVELOPMENT-SNAPSHOT-2021-03-02-a-osx.pkg) (March 2) has changed since I'm getting an error around `withUnsafeThrowingContinuation`.  I could see in the [forums](https://forums.swift.org/t/accepted-with-modification-se-0296-async-await/43318/20) some people were mentioning it.  Maybe they removed it?

**Reply by peterfriese** [reply]
Thanks, Edward!

I just installed the [most recent drop](https://swift.org/builds/development/xcode/swift-DEVELOPMENT-SNAPSHOT-2021-03-05-a/swift-DEVELOPMENT-SNAPSHOT-2021-03-05-a-osx.pkg) (March 5th) and was able to successfully compile and run the code. 

If you're able to repro, please feel free to file an issue with exact steps (and versions of the toolchain, Xcode, iOS, and Simulators used).

**Reply by ejstembler** [reply]
I just realized your build target is iOS, whereas I'm trying this in macOS.

**Comment by ferologics** [comment]
I'd like to see an example of how async/await (and actors) fits the Combine api model.
